co2eq/purchase/index.js

```diff
@@ -201,11 +202,8 @@ export function carbonEmissionOfLineItem(lineItem, countryCodeISO2, datetime) {
   if (!entry.intensityKilograms) {
     throw new Error(`Missing carbon intensity for purchaseType: ${identifier}`);
   }
-  if (!entry.year) {
```

Comment:
Have you verified that the CPI correction does not break after removing this for a type of purchase that does not have a reference year in the footprints.yml?

Reply:
good point